<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SOFAOpenIGTLINKIF</class>
 <widget class="qMRMLWidget" name="SOFAOpenIGTLINKIF">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>536</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>SOFA Data Directory</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="ctkDirectoryButton" name="SOFADataDirectoryButton">
       <property name="SlicerParameterName" stdset="0">
        <string>savingDirectory</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Model Node:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="qMRMLNodeComboBox" name="SOFAMRMLModelNodeComboBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="nodeTypes">
        <stringlist notr="true">
         <string>vtkMRMLModelNode</string>
        </stringlist>
       </property>
       <property name="hideChildNodeTypes">
        <stringlist notr="true"/>
       </property>
       <property name="addEnabled">
        <bool>false</bool>
       </property>
       <property name="removeEnabled">
        <bool>false</bool>
       </property>
       <property name="editEnabled">
        <bool>false</bool>
       </property>
       <property name="renameEnabled">
        <bool>false</bool>
       </property>
       <property name="interactionNodeSingletonTag">
        <string notr="true"/>
       </property>
       <property name="SlicerParameterName" stdset="0">
        <string>modelNode</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveModelButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Run the algorithm.</string>
       </property>
       <property name="text">
        <string>Save Model to SOFA Dir</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="OpenIGTLinkCollapsibleButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>OpenIGTLink connection</string>
     </property>
     <property name="collapsed">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="serverActiveCheckBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
        <property name="SlicerParameterName" stdset="0">
         <string>serverUp</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="OIGTLconnectionLabel">
        <property name="text">
         <string>   OpenIGTLink server - INACTIVE</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>130</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkDirectoryButton</class>
   <extends>QWidget</extends>
   <header>ctkDirectoryButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>SOFAOpenIGTLINKIF</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>SOFAMRMLModelNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
